ui/components/component-library/index.js

```diff
@@ -3,7 +3,7 @@ export { AvatarFavicon } from './avatar-favicon';
 export { AvatarIcon, AVATAR_ICON_SIZES } from './avatar-icon';
 export { AvatarNetwork, AVATAR_NETWORK_SIZES } from './avatar-network';
 export { AvatarToken } from './avatar-token';
-export { AvatarWithBadge } from './avatar-with-badge';
+export { AvatarWithBadge, BADGE_POSITIONS } from './avatar-with-badge';
```

Comment:
Hmm our convention is to use the component name in the const but that would make it quite long. I'm `AVATAR_WITH_BADGE_BADGE_POSITIONS` I think just `AVATAR_WITH_BADGE_POSTIONS` could also work. I'm leaning toward keeping our convention at the moment.